providers/os/config/config.go

```diff
@@ -115,5 +115,31 @@ var Config = plugin.Provider{
 				},
 			},
 		},
+		{
+			Name:    "vagrant",
+			Use:     "vagrant host",
+			Short:   "a Vagrant host",
+			MinArgs: 1,
+			MaxArgs: 1,
+			Flags: []plugin.Flag{
+				/*
+					// looks like we currently do not have sudo support for vagrant
+					// Should we support it?
```

Comment:
The current cnquery version does not support it:
```
cnquery run vagrant default --sudo -c 'machine.baseboard{*}'                                                                                                      
→ loaded configuration from /etc/opt/mondoo/mondoo.yml using source default
→ discover related assets for 1 asset(s)
→ resolved assets resolved-assets=1
Query encountered errors:
failed to create resource 'machine.baseboard': could not retrieve smbios info for platform: permission denied
machine.baseboard: no data available
```

Reply:
Is the same flag as for local not working?

```
{
	Long:        "sudo",
	Type:        plugin.FlagType_Bool,
	Default:     "false",
	Desc:        "Elevate privileges with sudo.",
	ConfigEntry: "sudo.active",
},
```